medusa/indexers/tmdb/tmdb.py

```diff
@@ -187,6 +187,16 @@ def search(self, series):
 
         return OrderedDict({'series': mapped_results})['series']
 
+    def _get_shows_countries(self, tmdb_id):
+        """Retrieve show's countries from TMDB
+
+        :param tmdb_id: The shows tmdb id
+        :return: A string with the show's countries
+        """
+        show_info = self._get_show_by_id(tmdb_id)['series']
```

Comment:
Your searching tmdb by tmdb id. Only you are not 100% sure the tmdb id came from tmdb. It's just in theory. In practice  a tmdb could only come from tmdb. 

For eample an imdb id, could come from tvdb (mapped from tvdbid) or tvmaze (mapped from tvdbid or tvmazeid).
If one of them maps it to a wrong id, you could request for an id that does not exist on TMDB.

Does that make sense?